jpasskit/src/main/java/de/brendamour/jpasskit/PKPass.java

```diff
@@ -32,422 +29,199 @@
 import de.brendamour.jpasskit.passes.PKGenericPass;
 import de.brendamour.jpasskit.passes.PKStoreCard;
 
-public class PKPass implements IPKValidateable {
+public class PKPass implements Cloneable, Serializable {
 
     private static final long serialVersionUID = -1727648896679270606L;
 
-    private static final int EXPECTED_AUTHTOKEN_LENGTH = 16;
+    protected int formatVersion;
```

Comment:
I use protected for 2 reasons:
1. Need at least `package` accessibility to access the fields from builder without setters.
2. If I don't specify access modifier, code style validators start "yelling" at me :)

So `protected` is a "win-win" solution for me.

Reply:
Why don't you configure these validators, or exlude this class from this rule?
Or at least leave a comment about it because it is confusing like this, noone will remember.